paper/paper.md

```diff
@@ -0,0 +1,161 @@
+to their peers.
+We divided up the content by topic into Jupyter notebooks for each lesson, with
+each lesson taking approximately 15-20 minutes to teach via live coding.
+Each lesson has a corresponding practice notebook with additional exercises on
+the same content taught in the lesson, but with different data or variables
+used.
+We hosted the curriculum notebooks in a public GitHub repository to facilitate
+collaborative development and peer review using pull requests.
+In the initial curriculum drafting phase, developers were assigned lesson and
+practice notebooks to write.
+Once the draft of a lesson was completed, the writer opened a pull request and
+asked for review from a different developer.
+The reviewer then provided feedback and approved the pull request to be merged
+into the main branch after the writer made any requested changes.
+This way, more than one person viewed each notebook before it could be
+incorporated into the public curriculum, which reduced mistakes and ensured
+higher quality content. <!-- TODO: cite U-M Carpentries paper for the development model once it's submitted to bioRxiv?-->
+While teaching from the curriculum at the first Data Science Summer Experience,
+instructors took notes on their experience and fixed any problems afterward.
+Maintainers continue to monitor the repository and resolve issues as they arise.
+
+TODO: recording videos for virtual flipped classroom
+
+
+# Curriculum
+
+
+
+## Learning Objectives
+
+
+
+## Course Content
+
+
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+
+
+## Experience of Use
+
+We have used this curriculum to teach Summer Experince and Club in person in 2019 and virtually in 2020.
+For the in person instances, we taught the curriculum through live coding, a tenchique we learned from [The Carpentries](https://carpentries.org/) where the instructor types and explains the code while the learners follow along in real time.
+For the virtual instances, we used a flipped classroom approach where the learners watch videos of instructors explaining the material through "live" coding and follow along. 
+For both in person and virtual instances, we had several facilitators present at each session to answer questions and help learners debug. 
+Furthermore, one or two facilitators were assigned to each project group to help learners define data analysis questions, develop and execute a data analysis plan, visualize and communicate their findings, and troubleshoot coding problems. 
+
+We credit the success of our curriculum not only to the skill of the instructors, but also to the way we organized and executed the lessons and project:
+1. The instructors and learners all used Google co-lab to write and execute code in Jupyter notebooks.
+We chose this option because learners do not have to install any programs to use Google co-lab and you can easily open and edit the Jupyter notebooks from GitHub.  
```

Comment:
```suggestion
We chose this option because learners do not have to install any programs to use Google co-lab and you can easily open and edit the Jupyter notebooks from GitHub.  When meeting in person, most learners use Google Chromebooks which have limited programming capabilities, but easy use of a web browser.
```